### ISSUE TITLE
multiline input label

### ISSUE THREAD

**Issue by rowanG077** [react-toolbox/react-toolbox#777]
Hey guys!

There is a problem if you have a label that is longer than 1 line. The label will overflow the input. 

regards

**Comment by acornejo** [comment]
That is unfortunate.

However such a use of floating labels is not covered by the material design spec, and it sounds like poor UX. Labels spanning two lined are specifically listed as things you should not do from the spec in various other components (bottomnavigation for example, see here https://material.google.com/components/bottom-navigation.html#bottom-navigation-style ).

**Comment by rowanG077** [comment]
Yeah I would rather avoid it also. Unfortunately on devices with less vertical real estate it is unavoidable for some use cases I have.

Are you against supporting this, if not I can look into it.

**Comment by acornejo** [comment]
Not against it at all. I am sure a PR is welcome (it's Javi's call). If it requires a lot of css changes I would suggest you base it from the css-next branch.

**Comment by javivelasco** [comment]
Agree! Sorry for the delayed response!
